src/message.js

```diff
@@ -0,0 +1,242 @@
+'use strict'
+
+const { CID } = require('multiformats/cid')
+const { loadSync } = require('protobufjs')
+const { resolve } = require('path')
+
+const maxPriority = Math.pow(2, 31) - 1
+
+const definitions = loadSync(resolve(process.cwd(), 'bitswap.proto'))
```

Comment:
Please use `path.join(__dirname, 'bitswap.proto')`

Reply:
I'm trying to avoid `__dirname` if later we move to ESM on Node 16. Any strong reason for this?

Reply:
If you move to ESM, you'd have access to `import.meta.url` which you could use to do this.

This will make the process "fail" if spawn from any other folder than the current folder.

Reply:
Done! Will be in the next push.